apps/src/templates/courseOverview/CourseScript.js

```diff
@@ -116,6 +117,9 @@ class CourseScript extends Component {
       selectedSection.unitId === id;
     const isAssigned = assignedToStudent || assignedByTeacher;
 
+    let unitPath = location.pathname.includes('teacher_dashboard')
```

Comment:
nit: it would be a bit more future-proof to check whether the path starts with `/teacher_dashboard`
```suggestion
    let unitPath = location.pathname.startsWith('/teacher_dashboard');
```